app/src/main/res/values/strings.xml

```diff
@@ -49,4 +49,10 @@
     <!-- test -->
     <string name="home_test_data_warning">Please wait.\n\nWallet is generating test data.</string>
 
+    <!-- create wallet -->
+    <string name="crypto_wallet_label1">A crypto wallet</string>
```

Comment:
Adding some context into the name of the property helps, also this looks line line#1 of a 2-line text, so something like create_wallet_info_line_1 might look more understandable. If it's a string specific to an activity/fragment I always add the name of the activity/fragment as a prefix to the resource name.